kuma/wiki/models.py

```diff
@@ -811,6 +809,10 @@ def revert(self, revision, user, comment=None):
         revision.created = datetime.now()
         revision.creator = user
         revision.save()
+        # TODO: change to primary key check instead of object comparison
```

Comment:
Seems like that would have been easier to add a `.pk` here than write the comment. :grin: